### ISSUE TITLE
Upgrade to VectorPipe 2.0

### ISSUE THREAD

**Issue by CloudNiner** [azavea/osmesa#163]
This will require an upgrade to GT 3.1 and Spark 2.4+, so maybe not entirely trivial. We'll want to test this on staging and ensure our ECS tasks continue to function properly.

**Comment by CloudNiner** [comment]
This isn't something we currently need to unblock known work, but is noted as a future potential blocker.

**Comment by CloudNiner** [comment]
We have a compiling version of osmesa built against VP2.0 + GT3.0 in #182. I'm going close this to account for that work and write a new issue to deal with testing / deploying these changes to our staging/prod environment since we have higher priority tasks to address now.
